packages/react/src/DataTable/Pagination.module.css

```diff
@@ -0,0 +1,138 @@
+.TablePagination {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  column-gap: 1rem;
+  width: 100%;
+  grid-area: footer;
+  padding: var(--base-size-8) var(--base-size-16);
+  border: var(--borderWidth-thin) solid var(--borderColor-default);
+  border-top-width: 0;
+  border-end-start-radius: var(--borderRadius-medium);
+  border-end-end-radius: var(--borderRadius-medium);
+}
+
+@media ((max-width: calc(768px - 0.02px))) {
```

Comment:
Completely separate from the refactor, would this be something we would ever want a container query for now that we can use them or is this best tied to the viewport range? (Was just curious)

Reply:
Yes I think this was a bit verbose and would love either media query variables here or container queries.